### PR TITLE
Feature/mutopbar

### PR DESCRIPTION
**Pull request by AntoineRiche33**
Add component to fix issue #5

### CODE REVIEW

File: app/src/main/java/com/rlab/sejima/features/MUTopBar.java [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+    /**
+     * Constructor with attributes
+     * @param context the view context
+     * @param attrs the XML attributes for the view
+     */
+    public MUTopBar(Context context, AttributeSet attrs) {
+        super(context, attrs);
+
+        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.MUTopBar);
```
[the remark this answers is not in the document]
Comment (AntoineRiche33) [reply app/src/main/java/com/rlab/sejima/features/MUTopBar.java]: Changed `a` to `attributes`

File: app/src/main/java/com/rlab/sejima/features/MUTopBar.java [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    }
+
+    public void setButtonImage(int buttonImage) {
+        mButtonImage = buttonImage;
+        Drawable drawable = null;
+        try {
+            drawable = getResources().getDrawable(mButtonImage);
+            mIBLeftButton.setImageDrawable(drawable);
+        } catch (Resources.NotFoundException e){
+            Log.e(getClass().getCanonicalName(), "Img drawable not found", e);
```
[the remark this answers is not in the document]
Comment (AntoineRiche33) [reply app/src/main/java/com/rlab/sejima/features/MUTopBar.java]: Just add it to keep trace of error. It's only for debug so we don't need it anymore.

File: app/src/main/java/com/rlab/sejima/features/MUTopBar.java [hunk 3]
```diff
@@ -101,25 +104,29 @@ public MUTopBar(Context context) {
...
         init(context);
-        a.recycle();
+        attributes.recycle();
     }
 
+    /**
+     *
+     * @return the left padding of image button
+     */
     public float getLeftButtonLeading() {
```
Comment (damien-nd) [inline app/src/main/java/com/rlab/sejima/features/MUTopBar.java]: Done for getLeftButtonLeading but still need documentation for:
- isButtonHidden()
- setButtonHidden(boolean hideButton)
- getMUTopBarClickListener()
- setMUTopBarClickListener(MUTopBarClickListener clickListener)

File: app/src/main/java/com/rlab/sejima/features/MUTopBar.java [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+        setTitleAlignment(mTitleAlignment);
+        addView(mTVLabel);
+    }
+
+
+    /**
+     * Method triggered each time button is tapped.
+     */
+    public void didClick(){
+        Log.d(this.getClass().getName(), "MUTopBar has been clicked 2");
```
[the remark this answers is not in the document]
Comment (AntoineRiche33) [reply app/src/main/java/com/rlab/sejima/features/MUTopBar.java]: I've added an interface to handle clicks.

**Pull request merged**